### PR TITLE
Bug/reset css override

### PR DESCRIPTION
**Pull request by VitaliiDenysiuk**
1) fix some stiles which was override reset.css
2) fix clean message tweet after send
3) fix format local date time

### CODE REVIEW

**Review by DavidKorystov — APPROVED** [review]

File: src/redux/reducers/tweets.ts [hunk 1]
```diff
@@ -151,14 +152,21 @@ const tweetsSlice = createSlice<TweetsStore, SliceCaseReducers<TweetsStore>>({
...
         // and only such way I resolved this problem
         store.feedTweets = {
           ...store.feedTweets,
           ...payload,
-          docs: [...store.feedTweets.docs, ...payload.docs],
+          docs: [
+            ...store.feedTweets.docs,
+            ...payload.docs.map((item) => ({
+              ...item,
+              createdAt: localDateTime(item.createdAt),
```
Comment (alexandrtovmach) [inline src/redux/reducers/tweets.ts]: it's incorrect way
you shouldn't change any data, that you received from backend before setting it to store. Redux store, it's kinda your backend DB on frontend. Instead, you should make formatting in render

File: src/styles/global.css [hunk 2]
```diff
@@ -14,3 +14,43 @@ body {
 .ql-editor p em {
   font-style: italic;
 }
+
+strong {
```
Comment (alexandrtovmach) [inline src/styles/global.css:18]: huh, but it's not what I've asked to do. Basically, what you did, is override of reset.css, which is probably fine, but hacky. Did you try to pass custom components to quill?

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: src/redux/reducers/tweets.ts (hunk 1 above)
Comment (VitaliiDenysiuk) [reply src/redux/reducers/tweets.ts]: fixed

File: src/styles/global.css (hunk 2 above)
Comment (VitaliiDenysiuk) [reply src/styles/global.css:18]: I decided that it was very bad if standard tag lost their meaning and return important part of their style for all leleka. ok i will check if i can change just for quill

Comment (VitaliiDenysiuk) [reply src/styles/global.css:18]: and we have to change style not only for quill. Quill only help styling editor block but all other tweets rendered without quill and all this style its had to have too.

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**